fhir-model/src/main/java/com/ibm/fhir/model/util/FHIRUtil.java

```diff
@@ -225,7 +225,8 @@ public static OperationOutcome buildOperationOutcome(Exception exception, IssueT
         Throwable e = exception;
         String causedBy = "";
         while (e != null) {
-            msgs.append(causedBy + e.getClass().getSimpleName() + ": " + (e.getMessage() != null ? e.getMessage() : "<null message>"));
```

Comment:
For consistency, do you want to surround "null message" with `&lt;` and `&gt;`?

Reply:
Good question... I'm not sure. I'll let others weight in too

Reply:
I opted to encode it